integration-tests/pom.xml

```diff
@@ -373,6 +373,7 @@
                 <module>micrometer-mp-metrics</module>
                 <module>micrometer-prometheus</module>
                 <module>opentelemetry</module>
+                <module>opentelemetry-quickstart</module>
```

Comment:
Do we really need a new module for this? Is there no existing module we can use?

Reply:
Unfortunately no, because all the existing apps try to inject OTel stuff to help with the tests (like Baggage) and they will fail if OTel is disabled.

Reply:
Some of the test apps could probably be refactored with this objective but this is not the first time a bug is found on the quickstart, therefore I thought it would be worthwhile.